daemon/web/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "inertia-web",
+  "version": "1.0.0",
+  "description": "Simple, self-hosted continuous deployment",
+  "author": "UBC Launch Pad",
+  "license": "MIT",
+  "main": "index.js",
+  "scripts": {
+    "start": "NODE_ENV=development webpack-dev-server --hot --host 0.0.0.0",
+    "build": "NODE_ENV=production webpack --config webpack.prod.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "dependencies": {
+    "babel-core": "^6.26.0",
+    "babel-loader": "^7.1.2",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "html-webpack-plugin": "^3.0.6",
+    "npm": "^5.7.1",
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
+    "update": "^0.7.4",
```

Comment:
why did we need the `npm` and `update` packages?

Reply:
This was based off the `package.json` of the ubclaunchpad/bumper project

Not sure what `update` does, but I'll try removing them and see if it affects the build

Reply:
I think it's probably safe to remove. https://www.npmjs.com/package/update

Reply:
Interesting, thanks for the heads up - removed it

npm dependencies 😨 😨